### PR TITLE
Add Documentation for Weatherflow Cloud Integration

### PR DESCRIPTION
**Pull request by jeeftor**
## Proposed change

Adding new documentation for the weatherflow cloud integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/106615
- Link to parent pull request in the Brands repository:  https://github.com/home-assistant/brands/pull/5035
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/weatherflow_cloud.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: WeatherFlow Cloud
+description: Instructions on the Cloud based WeatherFlow integration
+ha_release: '2024.1'
```
Comment (gjohansson-ST) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
ha_release: '2024.3'
```

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]

File: source/_integrations/weatherflow_cloud.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: WeatherFlow Cloud
+description: Instructions on the Cloud based WeatherFlow integration
+ha_release: '2024.3'
+ha_category:
+  - Environment
```
Comment (gjohansson-ST) [inline source/_integrations/weatherflow_cloud.markdown:6]:
```suggestion
ha_category:
  - Environment
  - Weather
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: WeatherFlow Cloud
+description: Instructions on the Cloud based WeatherFlow integration
+ha_release: '2024.3'
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow_cloud.markdown:4]:
```suggestion
ha_release: 2024.3
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ha_category:
+  - Environment
+  - Weather
+ha_platforms:
+  - weather
+ha_iot_class: Cloud Poll
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow_cloud.markdown:10]:
```suggestion
ha_iot_class: Cloud Polling
```

**Review by gjohansson-ST — APPROVED** [review]
Thanks @jeeftor 👍

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
What do users get when they add this integration to Home Assistant, which type of entities etc?

**Comment by jeeftor** [comment]
@klaasnicolaas  - right now its just the forecast entities ... but I'll start adding entities in now that its in since I've been keeping it "small"

**Comment by jeeftor** [comment]
>The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).

**Comment by gjohansson-ST** [comment]
@jeeftor please make a new PR to address the review comments I missed. Regarding the entities etc. I think it's not needed as it's limited in it's current scope and can perhaps be expanded if further things are added 👍

**Comment by klaasnicolaas** [comment]
It was a hint to add more information to the docs than just these 2 lines 😉 Could you adjust the other suggested changes in a new PR? Since this has (unfortunately) already been merged.

**Pull request merged**